src/snapred/backend/service/CalibrationService.py

```diff
@@ -109,6 +95,25 @@ def reduction(self, runs: List[RunConfig]):
                 raise
         return {}
 
+    @lru_cache
```

Comment:
what is `@lru_cache`?

Reply:
It creates a hash of the input values and caches the result of the request, so if its requested again it just returns the precomputed value.